recipes/MOABB/MotorImagery_decoding/BNCI2014001/MOABB_dataio_iterators.py

```diff
@@ -0,0 +1,409 @@
+import numpy as np
+from moabb.datasets.base import BaseDataset
+from moabb.paradigms.base import BaseParadigm
+from sklearn.model_selection import StratifiedKFold
+from itertools import islice
+import torch
+from torch.utils.data import TensorDataset, DataLoader
+import os
+
+
+def standardize(x, m, s):
+    """This function standardizes input EEG signals (x) using mean value (m)
+    and standard deviation (s)."""
+    demeaned = x - m.reshape((1, m.shape[0], 1))
+    standardized = demeaned / (1e-14 + s.reshape((1, s.shape[0], 1)))
+    return standardized
+
+
+def nth(iterable, n, default=None):
+    """This function returns cross-validation train and test indices of the i-th fold."""
+    return next(islice(iterable, n, None), default)
+
+
+class WithinSession(object):
+    """Within Session """
```

Comment:
Here you should improve the docstring with a better description of what we are doing + description  of the arguments (see other docstrings)